features/steps/notification_database.py

```diff
@@ -167,10 +167,14 @@ def insert_rows_into_new_reports_table(context):
 
 @given(u"I insert following row into table reported")
 @given(u"I insert following rows into table reported")
-def insert_rows_into_reported_table(context):
+def insert_rows_into_reported_table(context, report='', notified_at=None):
     """Insert rows into table reported."""
     cursor = context.connection.cursor()
 
+    notified_at_set = False
```

Comment:
it's used only on one line (187), so why not use

```
if notified_at is not None;
    notified_at = row["notified_at"]
```
instead?

Reply:
Yes, that looks much better. Will change that.

Reply:
Wait, no :D That's the same problem as I mentioned in Daniele's comment above, I think.

Reply:
Changed according to [this comment ](https://github.com/RedHatInsights/insights-behavioral-spec/pull/132#discussion_r997063920)